packages/ui/src/memberships/components/MemberProfile/MemberRoleToggle.tsx

```diff
@@ -62,9 +71,16 @@ export const MemberRoleToggle = ({ role }: MemberRoleToggleProps) => {
               <SidePaneRow>
                 <SidePaneLabel text="Earned total" />
                 <SidePaneColumn>
-                  <SidePaneText>
-                    <TokenValue value={earnings} />
-                  </SidePaneText>
+                  <EarnedTotalContainer>
+                    <SidePaneText>
+                      <TokenValue value={earnings} />
+                    </SidePaneText>
+                    {isLead && (
+                      <PayWorkerButton size="small" onClick={showPayWorkerModal}>
+                        Pay Worker
```

Comment:
Clicking on the button:
<img width="826" height="228" alt="invalid" src="https://github.com/user-attachments/assets/e9ef18ac-74a4-46cf-8684-3c1baeedf5dc" />